client/prototypes.h

```diff
@@ -25,11 +25,18 @@
 #include <openssl/md5.h>
 
 #define MAX_DIGEST_LENGTH 64
-typedef struct metalinkfile{
+
+typedef struct metalinkurls_t {
```

Comment:
the `_t` suffix convention is to indicate that a literal is a type in C. So please change as,

```
typedef struct metalinkfile {
    ...
} metalinkfile_t;
```

And use metalinkfile_t as your typename.

Reply:
Thanks, addressed.